### PR TITLE
Fix some metrics suffixes.

### PR DESCRIPTION
**Pull request by juliusv**
Gauges should end with a unit suffix (unless they are counts of
some object that is already mentioned in the metric name).

Counters should end with a _total suffix.

### CODE REVIEW

**Comment by juliusv** [comment]
@SuperQ @brian-brazil

File: mysqld_exporter.go [hunk 1]
```diff
@@ -205,7 +205,7 @@ var landingPage = []byte(`<html>
 // Metric descriptors for dynamically created metrics.
 var (
 	binlogSizeDesc = prometheus.NewDesc(
-		prometheus.BuildFQName(namespace, binlog, "size"),
+		prometheus.BuildFQName(namespace, binlog, "size_bytes"),
```
Comment (juliusv) [inline mysqld_exporter.go]: Actually, I'm not even sure if this is in bytes or not, which is exactly the problem - please verify.

File: mysqld_exporter.go [hunk 2]
```diff
@@ -349,79 +349,79 @@ var (
 				"The number of concurrent connections for this user.",
 				[]string{"user"}, nil)},
 		"CONNECTED_TIME": {prometheus.CounterValue,
-			prometheus.NewDesc(prometheus.BuildFQName(namespace, informationSchema, "user_statistics_connected_time"),
+			prometheus.NewDesc(prometheus.BuildFQName(namespace, informationSchema, "user_statistics_connected_time_total"),
```
Comment (brian-brazil) [inline mysqld_exporter.go]: Do we know the unit here?

File: mysqld_exporter.go (hunk 1 above)
Comment (brian-brazil) [inline mysqld_exporter.go]: It's bytes.

File: mysqld_exporter.go (hunk 2 above)
Comment (juliusv) [inline mysqld_exporter.go]: Oh right, same for the metrics below, they also need a time unit. I don't know it, hopefully @SuperQ does.

Comment (juliusv) [inline mysqld_exporter.go]: Ah, help string says seconds already, will update the metric name.

Comment (juliusv) [inline mysqld_exporter.go]: Updated to include `_seconds`.

**Comment by brian-brazil** [comment]
:+1:

**Comment by SuperQ** [comment]
:+1: Thanks for fixing.

**Pull request merged**